### DISCUSSION TITLE
IOS端消息排版显示异常

### DISCUSSION THREAD

**Discussion by liergazi** [Mrs4s/go-cqhttp#860] [Q&A]
从mirai-native转过来的，安卓和电脑显示正常
![1](https://user-images.githubusercontent.com/69339964/116298450-0f32fa00-a7cf-11eb-9a1d-3d004d925f4f.jpg)
![1619550507139](https://user-images.githubusercontent.com/69339964/116298461-10642700-a7cf-11eb-8ed4-e5938ee14137.png)

**Comment by Ink-33** [comment]
show your code
同一个消息，安卓/pc正常，ios异常。
这个问题本身就蛮诡异的

**Reply by liergazi** [reply]
是的。这个消息之前我是使用mirai-native发送的。那边是没有问题的。蛮诡异的

**Reply by Ink-33** [reply]
> 是的。这个消息之前我是使用mirai-native发送的。那边是没有问题的。蛮诡异的

你把消息复制出来看看是不是多了个换行符

**Reply by liergazi** [reply]
┌─────────┐
├[CQ:face,id=74]昵称:〔李二嘎子〕
├[CQ:face,id=187]身份:〔本群VIP〕
├─────────┤
├[CQ:face,id=158]余额:〔[CQ:emoji,id=120820][CQ:emoji,id=120815][CQ:emoji,id=120816][CQ:emoji,id=120819][CQ:emoji,id=120818]〕
├[CQ:face,id=192]奖券:〔[CQ:emoji,id=120812]〕
├─────────┤
├[CQ:face,id=190]皮肤:〔1P:[CQ:emoji,id=128309] 2P:[CQ:emoji,id=128308]〕
└─────────┘

**Comment by liergazi** [comment]
好 这个现象只在IOS设备上看得到 具体表现为face表情独占一行


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Mrs4s/go-cqhttp"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月28日(星期三) 凌晨3:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Mrs4s/go-cqhttp] IOS端消息排版显示异常 (#860)





 
这个问题你自己看着办, 我去群里看色图了

**Comment by liergazi** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年4月28日(星期三) 凌晨3:27
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Mrs4s/go-cqhttp] IOS端消息排版显示异常 (#860)





 
非常好, 接下来就是把iOS用户全部淘汰掉
